apps/server/src/utilities/hashString.ts

```diff
@@ -0,0 +1,6 @@
+import bcrypt from 'bcrypt';
+
+const { HASH_SALT_ROUNDS } = process.env;
+export const hashString = (string: string) => {
+    return bcrypt.hashSync(string, +HASH_SALT_ROUNDS!);
```

Comment:
Please use the async version of this method. Plus no need  null coercion I guess